cmake/usCTestScript_github.cmake

```diff
@@ -10,18 +10,24 @@ find_program(CTEST_GIT_COMMAND NAMES git)
 if(DEFINED ENV{BUILD_DIR})
   set(CTEST_DASHBOARD_ROOT $ENV{BUILD_DIR})
 else()
-  set(CTEST_DASHBOARD_ROOT "/tmp/us_builds")
+  if (NOT WIN32)
+    set(CTEST_DASHBOARD_ROOT "/tmp/us_builds")
+  else()
+    set(CTEST_DASHBOARD_ROOT "C:\\tmp\\us_builds")
```

Comment:
Either error out and tell the user to specify BUILD_DIR as a valid, writeable path